task-sdk/src/airflow/sdk/execution_time/task_runner.py

```diff
@@ -566,7 +566,9 @@ def _prepare(ti: RuntimeTaskInstance, log: Logger, context: Context) -> ToSuperv
 
 
 def run(
-    ti: RuntimeTaskInstance, log: Logger
+    ti: RuntimeTaskInstance,
+    context: Context,
+    log: Logger,
 ) -> tuple[IntermediateTIState | TerminalTIState, ToSupervisor | None, BaseException | None]:
     """Run the task in this process."""
```

Comment:
nit: arguments in doc-strings are missing.

Reply:
The entire execution time is internal and I don’t think we really need to describe these arguments to a contributor.

Reply:
That. Docs here are only useful if it helps Airflow core developers.